test/browser/component.js

```diff
@@ -513,8 +516,13 @@ describe(`Rendering exception`, function() {
   });
 
   it(`logs an error`, function() {
-    expect(el.logError.getCall(0).args[0]).to.contain(`Error while rendering`);
-    expect(el.logError.getCall(0).args[0]).to.contain(`breakable-app`);
+    expect(el.logRenderError.getCall(0).args[0]).to.contain(`Error while rendering breakable-app`);
```

Comment:
I'd say we can do away with this logging test now that there's a test for emitting an event on error. The useful part of this test wasn't to check that something would get written to the console, but rather that it would successfully catch a rendering error and not throw an exception.